e2e/stories/update-record.e2e.ts

```diff
@@ -0,0 +1,86 @@
+import * as faker from "faker";
+import * as Joi from "joi";
+import { field } from "../../src";
+import { DatasetRecordSchema } from "../lib/dataset";
+import { cleaning, DEFAULT_DATASET, dom, init } from "../teardowns";
+import { MESSAGE } from './../../src/config/message';
+
+const joiAssert = Joi.assert;
+
+jest.setTimeout(15000); // for the unstable internet connection
+
+describe("update record REST API", async () => {
+  const getDataSet = () => dom.dataset(DEFAULT_DATASET.NAME);
```

Comment:
I guess you don't need to call `dom.dataset()` in each test, it might be just:
```
const dataset = dom.dataset(DEFAULT_DATASET.NAME);
```
what do you think?

Reply:
done!